app/restore_to_file.go

```diff
@@ -126,11 +126,12 @@ func outputFormatSupported(desiredFormat string) bool {
 }
 
 func CheckBackingFileFormat(backingFilePath string) error {
-	output, err := exec.Command(QEMUImageBinary, "info", backingFilePath).Output()
+	output, err := exec.Command(QEMUImageBinary, "info", backingFilePath).CombinedOutput()
+	outStr := string(output)
 	if err != nil {
-		return err
+		return errors.Wrapf(fmt.Errorf(outStr), "Failed CheckBackingFileFormat %s", backingFilePath)
```

Comment:
There are a few issues with this statement (and below):
1. `err` was ignored instead of passing to the caller.
2. If you need to generate an error from scratch, use `fmt.Errorf` instead of `errors.Wrapf/Wrap`.
3. The change from `cmd.Output()` to `cmd.CombinedOutput()` has side effects. It's will contain the output to stderr in the second case. If that's what's needed, it's better to be done similar to https://github.com/rancher/longhorn-manager/blob/master/util/util.go#L216

To make it easier, you can create `util.ExecuteWithoutTimeout()` and use it for the command executions here (and below), e.g. (untested)
```
func ExecuteWithoutTimeout(binary string, args ...string) (string, error) {
       var err error
       cmd := exec.Command(binary, args...)

       var output, stderr bytes.Buffer
       cmd.Stdout = &output
       cmd.Stderr = &stderr

       if err = cmd.Run(); err != nil {
               return "", fmt.Errorf("Failed to execute: %v %v, output %s, stderr, %s, error %v",
                       binary, args, output.String(), stderr.String(), err)
        }
       return output.String(), nil
}
```